fargate.tf

```diff
@@ -0,0 +1,9 @@
+module "fargate" {
+  source                            = "./modules/fargate"
+  cluster_name                      = aws_eks_cluster.this[0].name
```

Comment:
```hcl
cluster_name = coalescelist(aws_eks_cluster.this[*].name, [""])[0]
```
Use splat syntax please to avoid this kind of problem https://github.com/terraform-aws-modules/terraform-aws-eks/pull/868

Reply:
Done